From f0e45cd42ba7e67ec444a4c14bac855b0ac195df Mon Sep 17 00:00:00 2001
From: "Tobias C. Berner" <tcberner@FreeBSD.org>
Date: Sat, 27 May 2017 13:54:56 +0200
Subject: [PATCH] Use ${X11_XTest_LIB} rather than Xtst.

Summary:
Use the cmake variable rather than the library name to link.
This fixes linking on FreeBSD, where the library location is not
in the default linker path.

Reviewers: #freebsd, davidedmundson

Reviewed By: davidedmundson

Subscribers: plasma-devel

Tags: #plasma

Differential Revision: https://phabricator.kde.org/D5997
---
 xembed-sni-proxy/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xembed-sni-proxy/CMakeLists.txt b/xembed-sni-proxy/CMakeLists.txt
index 1a5322a3..c2f67ac6 100644
--- xembed-sni-proxy/CMakeLists.txt
+++ xembed-sni-proxy/CMakeLists.txt
@@ -58,7 +58,7 @@ target_link_libraries(xembedsniproxy
     Qt5::DBus
     KF5::WindowSystem
     ${XCB_LIBS}
-    Xtst
+    ${X11_XTest_LIB}
 )
 
 install(TARGETS xembedsniproxy ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
-- 
2.13.0

